tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,8 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
+    assert result == False
```

Comment:
This assert is great for confirming the return value is correct, but what else could we assert to make sure there were no side affects like changes to the inventory?

Reply:
Perhaps I could add `assert vendor.inventory == ["a", "b", "c"]`?

Reply:
Yeah, that would work great! If for some reason we couldn't guarantee the order of the items (like if we stored them in a set), we could check that the length of the vendor inventory hasn't changed, then confirm each item that should be in the inventory is present either one by one or using something like the `all` syntax in the feedback on test_wave_06.py.